practices/respectful_code_reviews.md

```diff
@@ -0,0 +1,85 @@
+---
+slug: respectful-code-review
+stages:
+  - development
+short_description: Respectful code reviews involve giving a constructive feeback.The main goal should be to help deliver a good quality final product.
+tags:
+  - code review
+  - quality product
+  - communication
+  - conflict
+  - feedback
+  - knowledge sharing
+  - mentor
+  - comments
+  - code review
+keywords:
+  - code
+  - reviews
+  - respect
+  - constructive
+  - feedback
+  - quality
+  - comments
+  - communication
+---
+# Respectful Code Reviews
+
+**TL;DR**
+
+Respectful and constructive feedback is the basis of a fruitful collaboration, healthy work environment, and quality product.
+
+
+## Why You Might Want Respectful Code Reviews
+
+Code review helps to improve the quality of software projects but negative and unclear code review comments can have unwanted consequences.
+The right style of communication not only helps to build relationships and healthy work environments but also helps to achieve the final goal of delivering a quality product. It improves the overall health of the system and code.
```

Comment:
```suggestion
The right style of communication helps to build developer relationships and healthy work environments.  Respectful Code Reviews help to achieve the final goal of delivering a quality product. They improve the overall health of the system and code.```

Reply:
They improve the overall health of the system and code.```
I think extra characters have been added in the suggestion at the end

Reply:
Also, I meant  by relationships I meant workplace relationships between reviewer and developer(both may or may not be developers). Maybe, we can just keep the following point -The right style of communication helps to build  healthy work environments.  Respectful Code Reviews help to achieve the final goal of delivering a quality product. They improve the overall health of the system and code. 
If this helps clear confusion

Reply:
Also, by relationships I meant workplace relationships between reviewer and developer(both may or may not be developers). Maybe, we can just keep the following point -The right style of communication helps to build healthy work environments. Respectful Code Reviews help to achieve the final goal of delivering a quality product. They improve the overall health of the system and code.
If this helps clear confusion

Reply:
I see, use either your first version or the second (fact that I got confused doesn't mean it is not clear for 99 % of people)

Reply:
ok